pkgs/applications/science/biology/eggnog-mapper/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, makeWrapper, python27Packages, wget, diamond, hmmer }:
+
+python27Packages.buildPythonApplication rec {
+  pname = "eggnog-mapper";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "eggnogdb";
+    repo = "eggnog-mapper";
+    rev = "${version}";
+    sha256 = "1aaaflppy84bhkh2hb5gnzm4xgrz0rz0cgfpadr9w8cva8p0sqdv";
+  };
+
+
+
+  patches = [ ./add-scripts-to-setup.patch ];
```

Comment:
Why not use the commit directly from upstream?

```
(fetchpatch {
	url = https://github.com/eggnogdb/eggnog-mapper/pull/125/commits/b7828e4c8c1c453e391aef050f06ff3f84ff9faf.patch;
	sha256 = "...";
})
```

Reply:
Thanks. I didn't even know you could do that. Adapting it now...